cmd/doc/main.go

```diff
@@ -118,6 +130,18 @@ func main() {
 	start()
 }
 
+func getPageData(r *http.Request, disableNavBar bool) pageData {
+	var isDarkMode = false
+	if cookie, err := r.Cookie(cookieDarkMode); err == nil && cookie.Value == "dark-mode" {
```

Comment:
ah nice, so it will remember a user's preference? 🎉

Reply:
Exactly! It's actually possible for this to work without a server side but, but that was causing a brief flash so I figured I'd add it here.